sunpy/timeseries/metadata.py

```diff
@@ -208,7 +208,7 @@ def find_indices(self, time=None, colname=None, **kwargs):
         # Find all results with suitable timerange.
         results_indices = []
         for i, meta in enumerate(self.metadata):
-            if dt in meta[0] or not(dt):
+            if not(dt) or dt in meta[0]:
```

Comment:
I think you could drop the brackets here. i.e. `not dt or dt in meta[0]`

Reply:
How about `(not dt)`, just to be extra clear?